http/headers/feature-policy.json

```diff
@@ -0,0 +1,875 @@
+              "webview_android": {
+                "version_added": "60"
+              },
+              "chrome": {
+                "version_added": "60"
+              },
+              "chrome_android": {
+                "version_added": "60"
+              },
+              "edge": {
+                "version_added": false
+              },
+              "edge_mobile": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": "47"
+              },
+              "opera_android": {
+                "version_added": "47"
+              },
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          },
+          "meta-element-support": {
+            "__compat": {
+              "description": "<code>&lt;meta&gt;</code> element support",
+              "support": {
+                "webview_android": {
+                  "version_added": "60"
+                },
+                "chrome": {
+                  "version_added": "60"
+                },
+                "chrome_android": {
+                  "version_added": "60"
+                },
+                "edge": {
+                  "version_added": false
+                },
+                "edge_mobile": {
+                  "version_added": false
+                },
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": {
+                  "version_added": false
+                },
+                "ie": {
+                  "version_added": false
+                },
+                "opera": {
+                  "version_added": "47"
+                },
+                "opera_android": {
+                  "version_added": "47"
+                },
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": {
+                  "version_added": false
+                },
+                "samsunginternet_android": {
+                  "version_added": false
+                }
+              },
+              "status": {
+                "experimental": true,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "sandbox-attribute-support": {
```

Comment:
This should be `allow` attribute.

Reply:
Actually it's a cut and paste error. Node removed.